homeassistant/components/switch/rachio.py

```diff
@@ -8,7 +8,7 @@
 from homeassistant.components.switch import SwitchDevice, PLATFORM_SCHEMA
 from homeassistant.const import CONF_ACCESS_TOKEN
 
-REQUIREMENTS = ['rachiopy==0.1.2']
+REQUIREMENTS = ['rachiopy==0.1.1']
```

Comment:
But [0.1.2](https://pypi.python.org/pypi/RachioPy/0.1.2) is the one available on PyPI and seems to be the release created by the upstream project.